chainer/initializations.py

```diff
@@ -0,0 +1,120 @@
+from __future__ import absolute_import
+
+import numpy as np
+
+# Original code forked from MIT licensed keras project
+# https://github.com/fchollet/keras/blob/master/keras/initializations.py
+
+
+def init_weight(weights, initWith, scale=1,
+                none_default=lambda shape: he_normal(shape, 1 / np.sqrt(2))):
```

Comment:
My only concern is that a number of the interfaces in chainer use `None` to indicate a default weight initalizaton scheme. I would rather keep the `none_default` to better handle/matchup with the inputs being passed from the constructors. If you really want it the other way though I will change it.